src/render/TerraformRenderer.js

```diff
@@ -35,9 +35,10 @@ class TerraformRenderer extends DefaultRender {
   /**
    * Convert all provided components and links in terraform files.
    *
+   * @param {string} [parentEventId=null] - Parent event id.
```

Comment:
same

Reply:
It's manage by linter and jsdoc, if you set String we have a linter error. So it's good like that !